### ISSUE TITLE
Making Ebonweave, Spellweave, Moonshroud doesnt give buff

### ISSUE THREAD

**Issue by Zeorgbugfinder** [SallyWhitemane/Mograine#370]
**Description:**
title

**How to reproduce:**
make ebonweave

**How it should work:**
Making an Ebonweave should give 20crit for 30min, Spellweave 20haste, Moonshroud 10hp/mana regen

**Comment by Jamiexdx** [comment]
dont quote me on this but I believe it got removed at a later patch.

(3.3 so it couldn't get abused as a raidbuff)

**Comment by Zeorgbugfinder** [comment]
yes it was but on naxx/uld season where it requires specific territory and has 1day cd it should give 30min buff
